dockers/docker-auditd/config_checker.py

```diff
@@ -152,4 +153,6 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    while True:
+        main()
+        time.sleep(6000)
```

Comment:
Since this script is inside a container, it fits a container ENV, and your code just use as environment variable. The benefit is easy to change by container start parameters.

Reply:
no longer use loop in config_checker script

Reply:
Could you remove magic number and follow above suggestion in another PR?

Reply:
Sure, will address this comment in another PR